app/tests/conftest.py

```diff
@@ -0,0 +1,46 @@
+from unicodedata import name
+import pytest
+from app import create_app
+from app import db
+from flask.signals import request_finished
+from app.models.planet import Planet
+
+@pytest.fixture
+def saved_planets(app):
+    # Arrange
+    Uranus = Planet(
+        name="Uranus",
+        description="The third-largest planetary radius and fourth-largest planetary mass in the Solar System",
+        length_of_day="0d 17h 14m"
+        )
+    Earth = Planet(
+        name="Earth",
+        description="The most beautiful planet",
+        length_of_day="0d 0h 0m"
+        )
+
+    # db.session.add_all([Uranus, Earth])
+    # Alternatively, we could do
+    db.session.add(Uranus)
+    db.session.add(Earth)
+    db.session.commit()
```

Comment:
👍 

Nice work! My one critique is about `Uranus` and `Earth` variables looking like classes. Capitalizing the first letter in a name is reserved for class names like, `Planet`. To avoid confusion and follow pep8 guidelines, let's continue to practice lowercase for our variable names.